core/templates/components/certificate-assessment-offering-helper/certificate-offering-add-topic-items.component.html

```
@@ -0,0 +1,12 @@
+<div class="oppia-certificate-offering-add-topic-items">
+  <h2>Add Topics</h2>
+  <!--
+    Real topic search, available topics list, and selected topics list are
+    added with the topic-selection implementation.
+  -->
+
+  <div class="oppia-certificate-offering-add-topic-items-actions">
+    <button (click)="onBackClicked()">Back</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)